.env

```diff
@@ -194,3 +194,6 @@ REACT_APP_DISABLE_TASK_CLUSTERS=''
 
 # Disable User Leaderboard configurations
 REACT_APP_DISABLE_USER_LEADERBOARD_CONFIGS=''
+
+# Character requirement for challenge instructions
```

Comment:
The comment, conf, and all other vars should indicate it's a _minimum_ length. Otherwise the code reads as a max length.
So `REACT_APP_CHALLENGE_INSTRUCTIONS_MIN_LENGTH` or `REACT_APP_CHALLENGE_INSTRUCTIONS_MINIMUM_LENGTH`
and eg ` minLength: instructionsMinLength,`